common/web/gesture-recognizer/src/test/auto/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+/*
+ * VS Code Intellisense needs this helper in order to properly use subpath imports in the
+ * test specs found under the `headless` subfolder.  Otherwise, it'll report errors while
+ * editing - even if the tests themselves actually work.
+ */
+{
+  // Enabling the following line re-breaks it for some reason?
+  // "extends": "../../../tsconfig.json",
```

Comment:
This means that you are losing all the compiler settings for code in this folder so I'm not keen on this. It's a fix for an IDE issue that has impact on compilation, seems like a big step backwards.

One thing to check, make sure your IDE is using the right version of TypeScript:

![image](https://github.com/keymanapp/keyman/assets/4498365/f4b1fa03-1f2b-406c-b112-0d0b8ef4b864)

Reply:
OK, so the problem was that the `exclude` entry on the main project settings were preventing VS code from matching against the test .ts entries.  Looks like having them import settings from the main project's direct ancestor (for common/web TS settings) did the trick, since it doesn't define any `include` or `exclude` entries.